integrations/cohere/releasenotes/notes/add_cohere_chat_generator-ef96bbeaf588281c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+preview:
+  - |
+    Adds `CohereChatGenerator` for text and chat generation.
```

Comment:
We currently don't have release notes for this repository.
Please remove this file.